src/Entity/ItemId.php

```diff
@@ -45,7 +45,9 @@ private function assertValidIdFormat( $idSerialization ) {
 	}
 
 	/**
-	 * @return int
+	 * @see Int32EntityId::getNumericId
+	 *
+	 * @return int Guaranteed to be a unique integer in the range [1..2147483647].
```

Comment:
Oh sh*t merged too fast...

Don't think the "unique" part is correct. Or at least needs some very specific interpretation that is not specified. You can clearly have multiple instances with the same value. The uniqueness is only there in some context. (Which is why this class does not say anything about this in the first place)